### ISSUE TITLE
NAMLSS not working

### ISSUE THREAD

**Issue by muellermarkus** [OpenTabular/NAMpy#18]
Hey,

just wanted to let you know that the current NAMLSS implementation is not working. First off al, the dimensions do not add up, see https://github.com/OpenTabular/NAMpy/blob/7fbb144894cfa501a31a160b989aadfd588c7e50/nampy/basemodels/nam.py#L297

Changing this to `x = self.feature_dropout(torch.stack(all_outputs, dim=-1)).sum(dim=-1)` at least trains the model. However, I get other errors related to saving the parameters and deprecated functions.

**Comment by Ananyapam7** [comment]
Hey Marcus, sorry for the late reply. I am now maintaining this package. NAMLSS now works, let me know if you face any other issues.
